wiki/Tournaments/MP4K/3/en.md

```diff
@@ -56,29 +55,332 @@ The minimum guaranteed prizes are:
+| **F** | ![][flag_PL] SitekX | ![][flag_PL] Miq | ![][flag_PL] Felix27 | ![][flag_PL] Acrith |
+| **G** | ![][flag_PL] Tidek | ![][flag_PL] Hvb3rt | ![][flag_PL] Shadow Cultist | ![][flag_PL] FD5GD |
+| **H** | ![][flag_PL] \_underjoy | ![][flag_PL] Justin14p | ![][flag_PL] m666 | ![][flag_PL] MrBooM |
+
+---
+
+## Podium
+
+This competition has come to an end and resulted in the following podium:
+
+![MP4K3 Winner Podium](img/podium.png)
 
 ---
 
 ## Mappools
 
+### Grand Finals
+
+- FreeMod
+  - [Shaman Cure-All - Tuk Tuk Boshi (-Rem-) \[remuther\]](https://osu.ppy.sh/beatmapsets/1010796#mania/2115856)
+  - [DJ Sharpnel - Tactical Railroad Gamma (Kamikaze) \[Rush\]](https://osu.ppy.sh/beatmapsets/996504#mania/2084422)
+  - [Aquellex - Wanderflux (LeiN-) \[Luminous (MP4K3 Edit)\]](https://osu.ppy.sh/beatmapsets/1010684#mania/2115663)
+  - [keyc - blue traces (Kamikaze) \[\[\*>w<\*\]\]](https://osu.ppy.sh/beatmapsets/853515#mania/1783808)
+  - [LV.4 - Fragile (2011 Mix) (\[Crz\]Crysarlene) \[Lynessa's Handstream (225bpm)\]](https://osu.ppy.sh/beatmapsets/1007085#mania/2115843)
+  - [Tia - Deal with the devil (-Rem-) \[gamble\]](https://osu.ppy.sh/beatmapsets/995384#mania/2082239)
+  - [School Food Punishment - in bloom (MisterLuka) \[nanzen nanman\]](https://osu.ppy.sh/beatmapsets/884072#mania/1848003)
+  - [Kommisar - Turkey Turkey (Gekido-) \[LN Master\]](https://osu.ppy.sh/beatmapsets/852916#mania/1782630)
+  - [cosMo VS dj TAKA - Kyokken (Raveille) \[Nocturne\]](https://osu.ppy.sh/beatmapsets/895828#mania/1871803)
+  - [An - Catanoph (Toaph Daddy) \[Eternity\]](https://osu.ppy.sh/beatmapsets/696257#mania/1474927)
+  - [Terminal11 - Adcar (Gekido-) \[Challenge\]](https://osu.ppy.sh/beatmapsets/911470#mania/1902611)
+  - [Yuuyu - Scrap Syndrome (Halogen-) \[Manic Disassembly\]](https://osu.ppy.sh/beatmapsets/663256#mania/1403908)
+  - [xelloscope - Apollo (inteliser) \[UNLIMITED\]](https://osu.ppy.sh/beatmapsets/727736#mania/1536569)
+  - [Moe Shop w/ Ace Hashimoto - Natural (Percyqaz) \[going back for the gold\]](https://osu.ppy.sh/beatmapsets/784021#mania/1646203)
+  - [Graham Kartna - chiltonWALK.temp (Nivrad00) \[Challenge\]](https://osu.ppy.sh/beatmapsets/1010184#mania/2114659)
+- Tiebreaker  
+  - **[Quarks (kradness x Camellia) - Intro + Dualive (Evening) \[backpropogation\_edit\]](https://osu.ppy.sh/beatmapsets/996757#mania/2116122)**
+
+### Finals
+
+- FreeMod
+  - [yassu~ - before... (-Rem-) \[mizuki w/ remuther (MP4K3 Edit)\]](https://osu.ppy.sh/beatmapsets/1007040#mania/2107649)
+  - [The Dillinger Escape Plan with Mike Patton - Hollywood Squares (Pope Gadget) \[Noughts\]](https://osu.ppy.sh/beatmapsets/878577#mania/1837470)
+  - [YZYX - .\_Pulse (-Rem-) \[Challenge (240bpm\]](https://osu.ppy.sh/beatmapsets/1007048#mania/2107663)
+  - [guna with Team Kyukkyu - subconsciousness (-Rem-) \[Benzopirene w/ remuther (177bpm\)]](https://osu.ppy.sh/beatmapsets/1006457#mania/2106348)
```

Comment:
```suggestion
  - [guna with Team Kyukkyu - subconsciousness (-Rem-) \[Benzopirene w/ remuther (177bpm\)\]](https://osu.ppy.sh/beatmapsets/1006457#mania/2106348)
```
I'm not exactly sure why but it looks like the other `]` needs to be escaped

Reply:
oh lol

Reply:
Yes i noticed this, fixed on old commit